examples/flutter_infinite_list/lib/posts/bloc/post_bloc.dart

```diff
@@ -62,7 +61,8 @@ class PostBloc extends Bloc<PostEvent, PostState> {
 
   Future<List<Post>> _fetchPosts([int startIndex = 0]) async {
     final response = await httpClient.get(
-      'https://jsonplaceholder.typicode.com/posts?_start=$startIndex&_limit=$_postLimit',
+      Uri.https('jsonplaceholder.typicode.com', '/posts',
+          <String, String>{'_start': '$startIndex', 'limit': '$_postLimit'}),
```

Comment:
```suggestion
          <String, String>{'_start': '$startIndex', 'limit': '$_postLimit'},),
```
can we add a trailing comma and format?